services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/activities_scores_by_classroom_progress_report.jsx

```diff
@@ -52,28 +53,33 @@ export class ActivitiesScoresByClassroomProgressReport extends React.Component {
   }
 
   columns() {
+    const studentHeaderWidth = 360
     return ([
       {
         Header: 'Student',
         accessor: 'name',
         resizable: false,
         sortType: sortTableByLastName,
-        Cell: ({row}) => (<a className='row-link-disguise underlined' href={`/teachers/progress_reports/student_overview?classroom_id=${row.original.classroom_id}&student_id=${row.original.student_id}`}>
-          {row.original.name}
-        </a>),
+        Cell: ({ row }) => {
+          const { original } = row
+          const { id, name } = original
+          const link = `/teachers/progress_reports/student_overview?classroom_id=${row.original.classroom_id}&student_id=${row.original.student_id}`
```

Comment:
Again, should we extract `classroom_id` and `student_id` on the line above so that we can avoid `row.original` here?